CONTRIBUTING.md

```diff
@@ -75,19 +90,80 @@ to open an issue so we can add you to this list.
 
 # Coding Style
 
-labwc is written in Drew Devault's preferred [coding style] which is based
-on the [Linux kernel coding style].
+labwc is written in the [Linux kernel coding style] with a small number of
+deviations to align with [Drew Devault's preferred coding style] nameley:
+
+1. [Function Declaration](https://git.sr.ht/~sircmpwn/cstyle#function-declarations)
+2. [Braces for one-line statement](https://git.sr.ht/~sircmpwn/cstyle#brace-placement)
+3. [Organisation of header #include statements](https://git.sr.ht/~sircmpwn/cstyle#header-files)
+4. [Breaking of long lines](https://git.sr.ht/~sircmpwn/cstyle#splitting-long-lines)
 
 The reasons for specifying a style is not that we enjoy creating rules, but
 because it makes reading/maintaining the code and spotting problems much easier.
 
-If you're new to this style and want to get going quickly, either just imitate
+If you are new to this style and want to get going quickly, either just imitate
 the style around you, or read the summary below and use [checkpatch.pl] to run
 some formatting checks.
 
-## Style summary
+## Linux Kernel Style Basics
+
+The preferred line length limit is 80 columns, although this is a bit soft.
+
+Tabs are 8 columns wide. Identation with spaces is not used.
+
+Opening braces go on the same line, except for function definitions.
+
+Put `*` with the identifier when defining pointers, for example `char *foo`
+
+Spaces are placed around binary operators but not unary, like this:
+
+`sizeof(*foo)` is preferred over `sizeof(struct foo)`
+
+Use `if (!p)` instead of `if (p == 0)` or `if (p == NULL)`
+
+```
+int x = y * (2 + z);
```

Comment:
This code block belongs to "Spaces are placed around binary operators [..]"

Reply:
Thanks. Fixed.